collect_app/src/main/java/org/odk/collect/android/preferences/ServerPreferencesFragment.java

```diff
@@ -243,19 +244,28 @@ public void addOtherPreferences() {
     }
 
     public void initAccountPreferences() {
+        allowClickSelectedGoogleAccountPreference = true;
         accountsManager = new GoogleAccountsManager(this);
         accountsManager.setListener(this);
         accountsManager.disableAutoChooseAccount();
 
         selectedGoogleAccountPreference.setSummary(accountsManager.getSelectedAccount());
+
         selectedGoogleAccountPreference.setOnPreferenceClickListener(preference -> {
-            if (PlayServicesUtil.isGooglePlayServicesAvailable(getActivity())) {
-                accountsManager.chooseAccountAndRequestPermissionIfNeeded();
-            } else {
-                PlayServicesUtil.showGooglePlayServicesAvailabilityErrorDialog(getActivity());
+            if(allowClickSelectedGoogleAccountPreference){
+                allowClickSelectedGoogleAccountPreference = false;
+                if (PlayServicesUtil.isGooglePlayServicesAvailable(getActivity())) {
+                    accountsManager.chooseAccountAndRequestPermissionIfNeeded();
+                } else {
+                    PlayServicesUtil.showGooglePlayServicesAvailabilityErrorDialog(getActivity());
+                }
             }
             return true;
         });
+
```

Comment:
A few empty lines, probably that fails the build.

Reply:
yes i recommitted please check